pkgs/top-level/all-packages.nix

```diff
@@ -13873,6 +13873,10 @@ with pkgs;
     llvmPackages = llvmPackages_latest;
   };
 
+  crane = go-containerregistry.overrideAttrs (oldAttrs: rec {
+    subPackages = [ "cmd/crane" ];
+  });
```

Comment:
Why not copy the binaries to different outputs? That is friendlier for garbage collection and we only build once.

Reply:
I need to learn how to do that. Will research.